public/components/TagContext/TagRelationship/TagRelationship.react.js

```diff
@@ -100,6 +101,10 @@ export default class TagRelationship extends React.Component {
         return false;
       }
 
+      if (this.props.tag.type !== tagTypes.topic.name) {
```

Comment:
No, in this case it's fine, they're only being mapped to Keywords when they're crossing the boundaries to old systems.